content/posts/2024-02-06-der-sprudelfaktor-monatsbericht-januar-2024.en.md

```diff
@@ -0,0 +1,63 @@
+---
+sticky: false
+title: The Soda Factor – Monthly Review January 2024
+author: Community
+date: 2024-02-06T21:55:23.000Z
+draft: false
+categories:
+  - news
+  - monatsbericht
+  - dsee
+tags:
+  - umbau
+  - transparenz
+featured_image: /wp-content/uploads/2024/02/monatsbericht_naehcafe.jpg
+---
+
+Another month has passed, time to look back on January! With **18 public events**, one official visit to us and at least one away mission, we set a new record at the start of the year.
+
+The bi-weekly **Maker Mondays** continue to take place regularly, and on both dates in January they once again pulled crowds with 30 to 40 people working in the wood workshop, on the 3D printers or on the laser cutter. On January 8, Maker Monday also coincided with the **Sewing Café**, so that cutting and sewing took place at the front, while people milled and glued in the workshop at the back. The working group for **The Things Network/LoRaWAN** met three times for their open evenings. A playful approach to the principles of IT security was twice the topic for **Capture The Flag/PPP**, and five times we dealt with current computer game culture at **Live Let's Plays**.
```

Comment:
```suggestion
The bi-weekly **Maker Mondays** continue to take place regularly, and on both dates in January they once again pulled crowds with 30 to 40 people working in the wood workshop, on the 3D printers or on the laser cutter. On January 8, Maker Monday also coincided with the **Sewing Café**, so that cutting and sewing took place at the front, while people milled and glued in the workshop at the back. The working group for **The Things Network/LoRaWAN** met three times for their open evenings. A playful approach to the principles of IT security was twice the topic for **Capture The Flag/PPP**, and five times at [**Live Let's Plays**](https://defcon.social/@LLP222) among other things we dealt with which [basic needs](https://defcon.social/@LLP222/111716672458159793) a game has to fulfil , and how the m:f ratio for gamers looks like.
```

Reply:
```suggestion
The bi-weekly **Maker Mondays** continue to take place regularly, and on both dates in January they once again pulled crowds with 30 to 40 people working in the wood workshop, on the 3D printers or on the laser cutter. On January 8, Maker Monday also coincided with the **Sewing Café**, so that cutting and sewing took place at the front, while people milled and glued in the workshop at the back. The working group for **The Things Network/LoRaWAN** met three times for their open evenings. A playful approach to the principles of IT security was twice the topic for **Capture The Flag/PPP**, and five times we dealt with topics revolving around current computer game culture at [**Live Let's Plays**](https://defcon.social/@LLP222)
```